### DISCUSSION TITLE
What will happen when worker will die?

### DISCUSSION THREAD

**Discussion by lesnik512** [emmett-framework/granian#125] [Q&A] [answered]
Hi, uvicorn has a problem when master process stays alive when all workers die https://github.com/encode/uvicorn/issues/517

How granian behaves in such situation?

**Comment by gi0baro** [comment, marked as answer]
At the moment Granian doesn't check the state of worker processes, so it probably behaves the same as uvicorn.

**Reply by lesnik512** [reply]
thanks for the fast reply!
